src/stories/Library/cover/types.ts

```diff
@@ -4,7 +4,7 @@ export type CoverProps = {
   size: "xsmall" | "small" | "medium" | "large" | "xlarge";
   tint?: "20" | "40" | "80" | "100" | "120";
   coverUrl?: string;
-  description?: string;
-  shadow?: boolean;
+  alt?: string;
+  shadow?: boolean | "medium";
```

Comment:
Consider changing the type to only be a string. 

With the current setup it can both not be set, be false, be true, or be "medium". An alternate approach would be to make it optional (as now) and then have it accept one of a set of strings e.g. "small", "medium", whatever.

This might also make it easier to generate a control for the value in Storybook.

Reply:
Changed  the old boolean/default to "small" instead.